expressEngine/logger/location.js

```diff
@@ -0,0 +1,18 @@
+const getLocation = (stepInStack = 1) => {
+  try {
+    throw new Error('Log stack');
```

Comment:
Actually I want to know the location of file where log statement is used and I got the way is to use of stack and to read the stack I have to throw an error so that it can store it into stack.

Please let me know if I am not able to explain you and I will try again to my best.